src/Symfony/Component/Yaml/Parser.php

```diff
@@ -44,6 +48,16 @@ public function __construct($offset = 0, $totalNumberOfLines = null, array $skip
         $this->offset = $offset;
         $this->totalNumberOfLines = $totalNumberOfLines;
         $this->skippedLineNumbers = $skippedLineNumbers;
+        $this->tagResolver = new TagResolver();
```

Comment:
make this a constructor argument instead? (default null, no need for any instance when not using tags)